CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 3.3.2 (2020-03-16)
+
+- Fixed the autocomplete component to properly highlight results after removing search text. [#65](https://github.com/blackbaud/skyux-lookup/pull/65)
+- Fixed the autocomplete component to properly highlight results when only one character is supplied. [#65](https://github.com/blackbaud/skyux-lookup/pull/65)
+- Fixed the country field component to properly fire the `selectedCountryChange` event when used outside of a form. [#63](https://github.com/blackbaud/skyux-lookup/pull/63)
+- Fixed the lookup component to correctly validate required values within a template-driven form. [#64](https://github.com/blackbaud/skyux-lookup/pull/64)
```

Comment:
I'd delete "correctly." And I'd change "within a template-driven form" to "on template-driven forms" to match up with the plural "values."